godot-core/src/builtin/basis.rs

```diff
@@ -364,15 +364,19 @@ impl Basis {
         Self::from_cols(self.rows[0], self.rows[1], self.rows[2])
     }
 
-    /// Returns the orthonormalized version of the matrix (useful to call from
+    /// ⚠️ Returns the orthonormalized version of the matrix (useful to call from
```

Comment:
oh i wasn't aware of that distinction. fair enough.

Reply:
We can also discuss it of course. I thought it would be redundant if we already have a "Panics" section that had the exact same meaning. But maybe it helps in IDE suggestions... 🤷